gatsby-config.js

```diff
@@ -23,7 +23,7 @@ const contentfulConfig = {
 // To change back to the normal CDA, remove the CONTENTFUL_HOST variable from your environment.
 if (process.env.CONTENTFUL_HOST) {
   contentfulConfig.host = process.env.CONTENTFUL_HOST;
-  contentfulConfig.accessToken = process.env.CONTENTFUL_PREVIEW_ACCESS_TOKEN;
+  contentfulConfig.accessToken = process.env.CONTENTFUL_DELIVERY_TOKEN;
```

Comment:
You invert the logic users are used to:

If you have no host, we assume you want it from CDA (content delivery)

If you set a host, we assume you set it to CPA (content preview)

Reply:
Maybe change the way you set your environment variables?

Reply:
Thanks! I'll bring this up with the team and see what they suggest :+1:

Reply:
I've reverted this change for now, but am leaving this comment unresolved until I get some feedback from our product team regarding the environment variable change.

Reply:
👍